client/look-and-feel/react/src/Layout/Header/__tests__/Header.test.tsx

```diff
@@ -49,16 +60,27 @@ describe("Header Component", () => {
   });
 
   it("should hide burger menu on window resize above the breakpoint", async () => {
-    render(<Header rightItem={rightItem}>{children}</Header>);
-
     global.innerWidth = BREAKPOINT.MD - 1;
 
+    render(<Header rightItem={rightItem}>{children}</Header>);
+
     const burgerButton = screen.getByRole("button", {
       name: /Ouvrir le menu/i,
     });
 
     await userEvent.click(burgerButton);
 
-    expect(burgerButton).toHaveAttribute("aria-label", "Fermer le menu");
+    await waitFor(async () => {
+      const modalEl = screen.getByRole("dialog");
+      expect(modalEl).toHaveAttribute("open");
+    });
+
+    global.innerWidth = BREAKPOINT.MD + 1;
+    fireEvent(window, new Event("resize"));
+
+    await waitFor(async () => {
```

Comment:
Pas besoin d'async normalement
```suggestion
    await waitFor(() => {
```